### DISCUSSION TITLE
Osu lazer iOS 2022.226.0 not showing up on TestFlight

### DISCUSSION THREAD

**Discussion by Zyflx** [ppy/osu#17003] [Q&A] [answered]
Has anyone been able to get the new update on iOS because it’s no showing up for me.It still shows the old version.
![386755F2-6BE0-4143-9667-7CB7CEE888B9](https://user-images.githubusercontent.com/100444522/155864710-343b7bef-2d92-4de7-b30b-05ccec713ee4.png)

**Comment by peppy** [comment, marked as answer]
We pulled it due to issues. The next build will (hopefully) return.

**Reply by Zyflx** [reply]
Alright then thanks
